src/com/twitter/intellij/pants/service/PantsCompileOptionsExecutor.java

```diff
@@ -195,6 +234,41 @@ private ProcessOutput getProcessOutput(
   }
 
   @NotNull
+  private String getExport(
+    @NotNull String target,
+    @Nullable ProcessAdapter processAdapter
+  ) {
+    try {
+      final File outputFile = FileUtil.createTempFile("pants_export_" + target.replace('/', '_') + "_run", ".out");
```

Comment:
It's okay to have a static output file name, since it will be loaded before the next pants run and the next pants run will overwrite it.

If you really want to have a different name for each run, please see use `File.separator` instead of `/`. See `com.twitter.intellij.pants.service.PantsCompileOptionsExecutor#getProjectName` for example.